classes/class3/hw/views/catlistcolor.handlebars

```diff
@@ -0,0 +1,19 @@
+<center>
```

Comment:
You could have used the same template for `catlistcolor.handlebars` and `catlist.handlebars` with some more leveraging of Handlebars' capabilities.